eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/OnChainFeeConf.scala

```diff
@@ -18,25 +18,10 @@ package fr.acinq.eclair.blockchain.fee
 
 import fr.acinq.bitcoin.scalacompat.Crypto.PublicKey
 import fr.acinq.bitcoin.scalacompat.Satoshi
-import fr.acinq.eclair.blockchain.CurrentFeerates
 import fr.acinq.eclair.channel.{ChannelTypes, SupportedChannelType}
 import fr.acinq.eclair.transactions.Transactions
 
-trait FeeEstimator {
-  // @formatter:off
-  def getFeeratePerKb(target: Int): FeeratePerKB
-  def getFeeratePerKw(target: Int): FeeratePerKw
-  def getMempoolMinFeeratePerKw(): FeeratePerKw
-  // @formatter:on
-}
-
-case class FeeTargets(fundingBlockTarget: Int, commitmentBlockTarget: Int, commitmentWithoutHtlcsBlockTarget: Int, mutualCloseBlockTarget: Int, claimMainBlockTarget: Int, safeUtxosThreshold: Int) {
```

Comment:
We may still want to keep a slow/fast preference for funding and closing txs, that would fit in `OnChainFeeConf`. WDYT?

Reply:
Yes definitely, I think the node operator should be able to define in `eclair.conf` something like the following:

```
confirmation-target {
      // target for the funding transaction
      funding = slow/medium/fast
      // target for the commitment transaction (used in force-close scenario) *do not change this unless you know what you are doing*
      commitment = slow/medium/fast
      // target for the commitment transaction when we have no htlcs to claim (used in force-close scenario) *do not change this unless you know what you are doing*
      commitment-without-htlcs = slow/medium/fast
      // target for the mutual close transaction
      mutual-close = slow/medium/fast
      // target for the claim main transaction (tx that spends main channel output back to wallet)
      claim-main = slow/medium/fast
    }
```

Those were previously a block target, they should now be an enum `slow`, `medium` or `fast`?

Reply:
Done in 133264741.